File: docs/docs/API-Reference/python-examples/workflows-api/example-quickstart-sync.py
import os

import requests

base = os.environ.get("LANGFLOW_URL") or os.environ.get("LANGFLOW_SERVER_URL", "")
flow_id = os.environ.get("FLOW_ID", "")
api_key = os.environ.get("LANGFLOW_API_KEY", "")

headers = {"Content-Type": "application/json", "x-api-key": api_key}
payload = {
    "flow_id": flow_id,
    "input_value": "what is 2+2",
    "session_id": "session-123",
}

response = requests.post(f"{base}/api/v2/workflows", headers=headers, json=payload, timeout=120)
response.raise_for_status()
body = response.json()
print(body["output"]["text"])

File: docs/docs/API-Reference/javascript-examples/workflows-api/example-quickstart-sync.js
const base = (process.env.LANGFLOW_URL ?? process.env.LANGFLOW_SERVER_URL ?? "").replace(/\/$/, "");
const flowId = process.env.FLOW_ID ?? "";
const apiKey = process.env.LANGFLOW_API_KEY ?? "";

const response = await fetch(`${base}/api/v2/workflows`, {
  method: "POST",
  headers: {
    "Content-Type": "application/json",
    "x-api-key": apiKey,
  },
  body: JSON.stringify({
    flow_id: flowId,
    input_value: "what is 2+2",
    session_id: "session-123",
  }),
});

if (!response.ok) {
  throw new Error(`HTTP ${response.status}: ${await response.text()}`);
}

const body = await response.json();
console.log(body.output.text);
